src/api/server.rkt

```diff
@@ -229,6 +241,18 @@
         (hash-remove! waiting job-id)]
        ; Get the result for the given id, return false if no work found.
        [(list 'result job-id handler) (place-channel-put handler (hash-ref completed-work job-id #f))]
+       [(list 'timeline job-id handler)
+        (define wid (hash-ref current-jobs job-id #f))
+        (cond
+          [(not (false? wid))
```

Comment:
`(not (false? foo))` is just the same as `(true? foo)` which is just the same as `foo`, right?

Reply:
Ya. I guess I want this to be an Optional/Maybe type and not used to truthy, falsely.